ice40/yosys/synth.tcl

```diff
@@ -1,6 +1,8 @@
 yosys -import
 
-synth_ice40 -nocarry
+synth_ice40 -nocarry -noabc
+# TODO: revert when libblifparse fix propagates to vpr
```

Comment:
Link to an issue so we can track the work?

Reply:
Done #428